module-2/backend/week_one.md

```diff
@@ -5,30 +5,84 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+GET - used to grab an object from the database
+POST- used to post an object to the database
+UPDATE- update attributes 
+PATCH -update attributes
+DELETE- delete an object
+
 2. What is Sinatra?
+DSL for Ruby web applications
+
 4. What is MVC?
+Models, Views, Controllers -> helps implement a user interface on a web application
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+(Guess) If anyone were to look at our code and want to understand how the MVC works, it's best to follow convention so that they can understand the application.

```

Comment:
This is a pretty solid answer. I like to think it allows me (and colleagues) to jump into an application cold and start contributing quickly.